serverless/pytorch/shiyinzhang/iog/nuclio/function.yaml

```diff
@@ -23,40 +23,22 @@ spec:
 
   build:
     image: cvat.pth.shiyinzhang.iog
-    baseImage: continuumio/miniconda3
+    baseImage: ubuntu:22.04
 
     directives:
       preCopy:
-        - kind: WORKDIR
-          value: /opt/nuclio
-        - kind: RUN
-          value: conda create -y -n iog python=3.9
-        - kind: SHELL
-          value: '["conda", "run", "-n", "iog", "/bin/bash", "-c"]'
-        - kind: RUN
-          value: conda install -y -c anaconda curl
-        - kind: RUN
-          value: conda install -y pytorch torchvision -c pytorch
-        - kind: RUN
-          value: apt update && apt install -y libgl1-mesa-glx
         - kind: RUN
-          value: conda install -y -c conda-forge pycocotools scipy
+          value: apt update && apt install -y libgl1-mesa-glx libglib2.0-0 python-is-python3 python3-pip git && rm -rf /var/lib/apt/lists/*
         - kind: RUN
-          value: pip install opencv-python gdown
+          value: pip install torch==2.9.0 torchvision==0.24.0 opencv-python gdown pycocotools scipy --extra-index-url https://download.pytorch.org/whl/cpu --no-cache-dir
```

Comment:
Would `opencv-python-headless` work here? That would remove the requirement to install `libgl1-mesa-glx libglib2.0-0`.